pom.xml

```diff
@@ -768,7 +769,7 @@
             <dependency>
                 <groupId>org.apache.avro</groupId>
                 <artifactId>avro</artifactId>
-                <version>1.11.1</version>
+                <version>${avro.version}</version>
```

Comment:
Really not important, but is there a specific reason for using a property for the version number here?

Reply:
Ah there's no specific reason. I just thought using a property would be the standard approach?